src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/response/UserTransactionActionResponse.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.sandl.snlevents.model.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserTransactionActionResponse {
+    UUID transactionId;
+    String action;
+    boolean success;
```

Comment:
Instead of having a boolean, do you thing it would make sense to use the UserTransactionStatus here? It has keys like COMMITED, ROLLEDBACK. Adding another key like: FAILED would do the job? I'm not truly conviced to what I proposed, trying to start a discussion

Reply:
On one hand, I would stay with a boolean to just inform of whether the action is ok or not and leave to the client what to do about it (ask for more details or react in a different way).
On the other hand, it may be beneficial to the client to know the reason of failure straight away, but that would mean the system should add something like that now and currently we still gonna deal with it in one way.

Unless somebody already knows about potential different use cases of this, I would leave it as is.